gapic/templates/testing/_default_constraints.j2

```diff
@@ -1,7 +1,6 @@
 {% from '_pypi_packages.j2' import pypi_packages %}
 # This constraints file is required for unit tests.
 # List all library dependencies and extras in this file.
-google-api-core
```

Comment:
Revert once https://github.com/googleapis/python-api-core/pull/638 is merged and released to PyPI

Reply:
Will revert once https://github.com/googleapis/python-api-core/pull/646 is merged

Reply:
Done